component/Input/input.html

```diff
@@ -84,11 +84,29 @@ <h2>Validation form</h2>
                     sandbox="allow-scripts allow-same-origin">
                     </iframe>
                 </div>
+                <h2>Validation form with error</h2>
+                <div class="component-display-container">
+                    <form class="input-form">
+                        <label for="email-input" class="label-font-size label-color">Enter your email*</label>
+                        <input type="email" placeholder="example@xyz.com" class="form-input-box error-input" id="email-input">
+                        <label for="password" class="label-font-size label-color">Enter your password*</label>
+                        <input type="password" minlength="8" class="form-input-box password  error-input">
+                        <button class="button form-textarea-btn form-btn">Login</button>
```

Comment:
Since, the button is a part of form you need to give type='submit' attribute to it.